server/src/main/java/org/opensearch/cluster/routing/allocation/ExistingShardsAllocator.java

```diff
@@ -60,6 +60,13 @@ public interface ExistingShardsAllocator {
         Setting.Property.PrivateIndex
     );
 
```

Comment:
Java doc here, we are adding a new public setting. Highly recommend adding multi-line comment explaining the purpose of the setting.